docs/ref/templates/builtins.txt

```diff
@@ -961,20 +961,20 @@ output (as a string) inside a variable. This is useful if you want to use
 
 Outputs a URL-encoded formatted query string based on the provided parameters.
 
-This tag requires a :class:`~django.http.QueryDict` instance, which defaults to
-:attr:`request.GET <django.http.HttpRequest.GET>` if none is provided.
+This tag requires a :class:`~django.http.QueryDict` or :class:`dict` instance,
+which defaults to :attr:`request.GET <django.http.HttpRequest.GET>`
+if none is provided.
 
-If the :class:`~django.http.QueryDict` is empty and no additional parameters
-are provided, an empty string is returned. A non-empty result includes a
-leading ``"?"``.
+If the ``dict``s are empty and no additional parameters are provided,
+an empty string is returned. A non-empty result includes a leading ``"?"``.
 
 .. admonition:: Using ``request.GET`` as default
 
     To use ``request.GET`` as the default ``QueryDict`` instance, the
     ``django.template.context_processors.request`` context processor should be
     enabled. If it's not enabled, you must either explicitly pass the
-    ``request`` object into the template context, or provide a ``QueryDict``
-    instance to this tag.
+    ``request`` object into the template context, or provide a
+    ``QueryDict``/``dict`` instance to this tag.
```

Comment:
Shouldn't we mention here that users can also pass `dict`s?

Reply:
I feel like this bit is about having the `request.GET` `QueryDict` being used as the default and so that you can pass a dict is, to me, not relevant for this note (if that makes sense)

Reply:
Ok yes this makes sense, I'm changing it.